# response of a robot, if it wants to take a certain task and if so what its score is

uint32 request_id
string robot_id
uint32 start_id
uint32 end_id
bool reject
float64 score
float64 estimatedDuration
